.gitignore

```
@@ -9,6 +9,9 @@ src/examples/ipc-interface.xml
 win*/binaries/
 win32/nsis/xiphos*.exe
 win32/po/messages.mo
+cmake/source_version.txt
+xiphos*.exe
+xiphos*.tar*
```

```suggestion
xiphos-*.exe
xiphos-*.tar*
```

A minor thing, but is more specific to the release artifacts (which we expect) and if we ever ended up with a stray *xiphos.exe* in the repo root (which we do not expect) it would show up as not belonging.

---

I would disagree on this point. We don't expect it to be there, but "git st" shouldn't be your mechanism for finding things that don't belong. `.gitignore` is just to be sure that it doesn't land in the git tree by accident. I prefer broad git ignore files, because there have been many times I've found stray generated files making their way into the repo that shouldn't have.

---

Sure.